sdk-lib/src/main/java/cash/z/ecc/android/sdk/Synchronizer.kt

```diff
@@ -605,6 +605,21 @@ interface Synchronizer {
      */
     suspend fun getTransactions(accountUuid: AccountUuid): Flow<List<TransactionOverview>>
 
+    /**
+     * Generates and returns an ephemeral address for one-time use, such as when receiving a swap
+     * from a decentralized exchange.
+     */
+    suspend fun getSingleUseTransparentAddress(accountUuid: AccountUuid): SingleUseTransparentAddress
+
+    /**
+     * Checks to find any single-use ephemeral addresses exposed in the past day that have not yet
+     * received funds, excluding any whose next check time is in the future. This will then choose the
+     * address that is most overdue for checking, retrieve any UTXOs for that address over Tor, and
+     * add them to the wallet database. If no such UTXOs are found, the check will be rescheduled
+     * following an expoential-backoff-with-jitter algorithm.
```

Comment:
Non-blocking: We should probably have documentation here on how an SDK user should be using this method, but we can add that later.